docs/data-sources/organizational_unit.md

```diff
@@ -15,21 +15,28 @@ Gets information about an organizational unit.
 ```terraform
 data "aiven_organizational_unit" "example_unit" {
   name      = "Example organizational unit"
-  parent_id = aiven_organization.main.id
 }
 ```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
```

Comment:
This looks a little suspicious. As we lost `name` as required attribute and we do not have `id` in this data source